src/main/resources/liquibase.properties

```diff
@@ -1,4 +1,4 @@
 url=jdbc:mysql://localhost/bookstoreDB?serverTimezone=UTC

```

Comment:
actually, I think you do not even need this file, all of those properties could be set in the general`application.prperties` file